ansible/roles/candlepin/tasks/el8/el8_base_deps.yml

```diff
@@ -0,0 +1,113 @@
+---
+- name: "Disable SELinux"
+  become: true
+  ansible.posix.selinux:
+    state: disabled
+  tags:
+    - system
+    - candlepin
+
+# FIXME: Update this task to use the dnf module once Ansible 2.10 or newer is readily available
+- name: "Update system packages"
+  become: true
+  ansible.builtin.command:
+    cmd: dnf update -y --allowerasing
+    warn: false
+  changed_when: false
+  tags:
+    - system
+    - system_update
+    - candlepin
+
+- name: "Enable required DNF modules"
+  become: true
+  ansible.builtin.command:
+    cmd: dnf module enable -y pki-core pki-deps
+    warn: false
```

Comment:
I just hit this on F37: Leaving the "warn" parameter in place actually breaks this task on newer versions of ansible, so we should probably drop these as we migrate over.

```
fatal: [el8]: FAILED! => {"changed": false, "msg": "Unsupported parameters for (ansible.legacy.command) module: warn. Supported parameters include: stdin_add_newline, _uses_shell, creates, removes, executable, argv, stdin, _raw_params, strip_empty_ends, chdir."}
```

There are a handful of these throughout that need to be trimmed.

Reply:
Done.

Regarding the readme. I copied the role readme but there's also a playbook readme in ansible folder.